concepts/pipeline/data_pipeline_detail.md

```diff
@@ -0,0 +1,429 @@
+
+## Self Serve Data Analysis
+
+```mermaid
+graph TD
+  subgraph Storage
+    lake[Data Lake]
+    s3_output_public[Public Outputs]
+    s3_output_private[Analysis Outputs]
+  end
+  subgraph ATMO
+    Jupyter -->|runs on| emr_cluster[Ad hoc EMR Cluster]
+    Zeppelin -->|runs on| emr_cluster
+    atmo_service[ATMO Service] -->|launch| emr_cluster
+    atmo_service -->|schedule| emr_job[fa:fa-clock-o Scheduled EMR Job]
+    emr_cluster -->|mount| EFS
+    emr_cluster --> lake
+    emr_job --> s3_output_public
+    emr_job --> s3_output_private
+  end
+  subgraph STMO
+    redash[Re:dash] -->|Presto| lake
+  end
+  subgraph TMO
+    evo[Evolution Dashboard]
+    histo[Histogram Dashboard]
+    agg[Telemetry Aggregates]
+    evo -.- agg
+    histo -.- agg
+  end
+  subgraph Databricks
+    db_notebook[Notebook]
+    db_notebook -->|Spark SQL| lake
+  end
+```
+
+Most of the data analysis tooling has been developed with the goal of being "self-serve". This means that people should be able to access and analyze data on their own, without involving data engineers or operations. Thus can data access scale beyond a small set of people with specialized knowledge of the entire pipeline.
+
+The use of these self-serve tools is described in the [Getting Started] article. This section focuses on how these tools integrate with the platform infrastructure.
+
+##### ATMO: Spark Analysis
+
+[ATMO] is a service for managing Spark clusters for data analysis. Clusters can be launched on demand, or can be scheduled to run a job on an ongoing basis. These clusters can read from the Data Lake described in the **Storage** section above, and can write results to either public (web-accessible) or private output locations.
+
+Jupyter or Zeppelin notebooks are the usual interface to getting work done using a cluster, though you get full SSH access to the cluster.
+
+Clusters launched via ATMO are automatically killed after a user-defined period of time (by default, 8 hours), though their lifetime can be extended as needed. Each cluster has an **`EFS`** volume mounted on the `/home/hadoop` directory, which means that data stored locally on the cluster persists from one cluster to the next.
+
+##### STMO: SQL Analysis
+
+[STMO] is a customized [Re:dash] installation that provides self-serve access to a a variety of different [datasets](/concepts/choosing_a_dataset.md). From here, you can query data in the Parquet Data Lake as well as various RDBMS data sources.
+
+STMO interfaces with the data lake using both [Presto] and Amazon [Athena]. Each has its own data source in Re:dash. Since Athena currently lacks the ability to use custom user-defined functions, access to datasets with HyperLogLog columns, such as [`client_count`](/datasets/batch_view/client_count/intro.md) are only available via Presto.
```

Comment:
Suggested revision:

Since Athena does not support user-defined functions, datasets with HyperLogLog columns, such as [`client_count`](/datasets/batch_view/client_count/intro.md), are only available via Presto.